packages/make-angular-cli-faster/src/utilities/nx-cloud.ts

```diff
@@ -1,9 +1,9 @@
-import { getPackageManagerCommand } from '@nrwl/devkit';
 import { execSync } from 'child_process';
 import { prompt } from 'enquirer';
+import { PackageManagerCommands } from 'nx/src/utils/package-manager';
 
-export async function promptForNxCloud(): Promise<boolean> {
-  const { useNxCloud } = await prompt<{ useNxCloud: 'Yes' | 'No' }>([
```

Comment:
Why did this change? Can you revert this please? If this is to get rid of the `tslib` then you can disable the `importHelpers` tsconfig option.